demos/ag-alert-keyvault-advisor/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {},
+  "variables": {
+    "actionGroupName": "myAcGroup"
+  },
+  "resources": [
+    {
+        "type": "microsoft.insights/actionGroups",
+        "apiVersion": "2021-09-01",
+        "name": "[variables('actionGroupName')]",
+        "location": "Global",
+        "properties": {
+            "groupShortName": "[variables('actionGroupName')]",
+            "enabled": true
+        }
+    }
+  ],
+  "outputs": {
+        "applicationGatewayResourceGroup": {

```

Comment:
you'll need to update these to match the parameter changes in the main template...